upgradability/set_code_hash/efficient_address_store/lib.rs

```diff
@@ -0,0 +1,97 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use ink_lang as ink;
+
+#[ink::contract]
+mod efficient_address_store {
+    use ink_prelude::vec::Vec;
+    use ink_env::set_code_hash;
+
+    #[derive(Default)]
+    #[ink(storage)]
+    pub struct EfficientAddressStore {
+        addresses: Vec<AccountId>,
```

Comment:
@fbielejec so this `Vec` is just the same thing as `std::Vec` and is not specially crafted for contracts? In other words: the whole vector is stored under a single key in the storage? I presume the answer is yes, but just wanted to make sure :P

Reply:
its coming from ink_prelude , because it's supposed to work with storage but indeed it occupies one key and read to [mem](https://docs.rs/ink_prelude/latest/ink_prelude/vec/index.html) in a whole. They have some plans to change that, but that's the situation rn:

Reply:
Thanks!